Dockerfile-release.riscv64

```diff
@@ -0,0 +1,13 @@
+FROM --platform=linux/ppc64le riscv64/debian:sid-slim
```

Comment:
New releases were migrated to gcr.io/distroless/static-debian11

Don't think we want to have use different images.

Reply:
Previously I saws lots of CVEs, so eventually we migrated to distroless base image.

Reply:
We don't want to use busybox due to licensing issues https://github.com/etcd-io/etcd/issues/15034

Reply:
It looks like `FROM scratch` just works.